reward/stakingInfo_test.go

```diff
@@ -72,25 +76,29 @@ func TestStakingInfo_GetStakingAmountByNodeId(t *testing.T) {
 	testCases := []struct {
 		address       common.Address
 		stakingAmount uint64
+		errorExist    bool
 	}{
-		{common.StringToAddress("0xB55e5986b972Be438b4A91d6e8726aA50AD55EDc"), 100},
-		{common.StringToAddress("0xaDfc427080B4a66b5a629cd633d48C5d734572cA"), 200},
-		{common.StringToAddress("0x994daB8EB6f3FaE044cC0c9a0AB1A038e136b0B6"), 300},
-		{common.StringToAddress("0xD527822212Fded72c5fE89f46281d5355BD58235"), 400},
-		{common.StringToAddress("0x027AbB8c9f952cfFf01B1707fF14E2CB5D439502"), 0},
+		{common.StringToAddress("0xB55e5986b972Be438b4A91d6e8726aA50AD55EDc"), 100, false},
+		{common.StringToAddress("0xaDfc427080B4a66b5a629cd633d48C5d734572cA"), 200, false},
+		{common.StringToAddress("0x994daB8EB6f3FaE044cC0c9a0AB1A038e136b0B6"), 300, false},
+		{common.StringToAddress("0xD527822212Fded72c5fE89f46281d5355BD58235"), 400, false},
+		{common.StringToAddress("0x027AbB8c9f952cfFf01B1707fF14E2CB5D439502"), 0, true},
 	}
 
-	stakingInfo, error := newEmptyStakingInfo(0)
-	if error != nil {
-		t.Errorf("error has occurred. error : %v", error)
-		t.FailNow()
-	}
+	stakingInfo := newEmptyStakingInfo(0)
 	stakingInfo.CouncilNodeAddrs = testdata.address
 	stakingInfo.CouncilStakingAmounts = testdata.stakingAmount
 
 	for i := 0; i < len(testCases); i++ {
-		result := stakingInfo.GetStakingAmountByNodeId(testCases[i].address)
+		result, err := stakingInfo.GetStakingAmountByNodeId(testCases[i].address)
+		var errExist bool
+		if err != nil {
+			errExist = true
+		} else {
+			errExist = false
+		}
 		assert.Equal(t, testCases[i].stakingAmount, result)
+		assert.Equal(t, testCases[i].errorExist, errExist)
```

Comment:
How about `assert.Equal(t, testCases[i].error, err)`? :)
- true -> ErrAddrNotInStakingInfo
- false -> nil

Reply:
reflected it